pom.xml

```diff
@@ -933,7 +933,7 @@
                 <module>persistence-modules/spring-data-neo4j</module>
                 <module>parent-boot-3</module>
                 <module>spring-kafka-2</module>
```

Comment:
Check my comment on
https://team.baeldung.com/browse/JAVA-25373

Reply:
@anastasiosioannidis I have added these in https://team.baeldung.com/browse/JAVA-26078